exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/EntityClass.kt

```diff
@@ -77,7 +78,9 @@ abstract class EntityClass<ID : Comparable<ID>, out T: Entity<ID>>(val table: Id
     fun removeFromCache(entity: Entity<ID>) {
         val cache = warmCache()
         cache.remove(table, entity)
-        cache.referrers.remove(entity.id)
+        refDefinitions.keys.forEach {
```

Comment:
Looks like this is meant to be a cache of references that create this entity (i.e. inbound).  Need to figure out how to make an outbound reference cache.